.github/workflows/stale.yml

```
@@ -0,0 +1,19 @@
+name: Mark stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3.0.7
+      with:
+        repo-token: ${{ secrets.BEDEVERE_GH_TOKEN }}
+        stale-pr-message: 'This PR is stale because it has been open for 30 days with no activity. Remove stale label or comment or this will be closed in 5 days'
+        stale-pr-label: 'stale-pr'
```

We already have a [stale label](https://github.com/python/cpython/labels/stale).

```suggestion
        stale-pr-label: 'stale'
```